src/main/java/core/Main.java

```diff
@@ -0,0 +1,42 @@
+package core;
+
+import core.service.ProcessingService;
+import core.service.ReaderService;
+import core.service.ReportHandlingService;
+import core.service.impl.ProcessingServiceImpl;
+import core.service.impl.ReaderServiceImpl;
+import core.service.impl.ReportHandlingServiceImpl;
+import core.strategy.FruitStrategy;
+import core.strategy.impl.BalanceOperation;
+import core.strategy.impl.PurchaseOperation;
+import core.strategy.impl.ReturnOperation;
+import core.strategy.impl.SupplyOperation;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+public class Main {
+    private static final String INPUT_FILEPATH = "src/main/java/resources/input.csv";
+
+    public static void main(String[] args) {
+        Map<String, FruitStrategy> strategyMap = createHashMapOperations();
+
+        ReaderService readerService = new ReaderServiceImpl();
+        List<String> queries = readerService.readFrom(INPUT_FILEPATH);
+
+        ProcessingService processingService = new ProcessingServiceImpl();
+        processingService.process(strategyMap, queries);
+
+        ReportHandlingService report = new ReportHandlingServiceImpl();
+        report.createReportLine();
```

Comment:
Maybe it's better to invoke writeTo() method explicitly and provide a file path here? 
This way we will know where the report will be stored and change destination place if needed
Just an option :)